resources/dicts/thoughts/traits/medicine.json

```diff
@@ -27,42 +32,49 @@
         "Is ripping some leaves to shreds",
         "Debates becoming a warrior",
         "Gives the wrong herbs to a warrior on purpose",
-        "Wishes they weren't confined to this life",
+        "Wishes {PRONOUN/m_c/subject} weren't confined to this life",
         "Is stocking up on deathberries",
         "Is stocking up on yew berries",
         "Is stocking up on nightshade",
         "Is stocking up on watermint",
         "Put herbs in prey to give cats stomachaches",
         "Refused to treat anyone today",
-        "Is upset that they can't fight alongside the warriors",
+        "Is upset that {PRONOUN/m_c/subject} can't fight alongside the warriors",
         "Wishes to fight alongside the warriors",
-        "Thinks they'd be an amazing leader",
+        "Thinks {PRONOUN/m_c/subject}'d be an amazing leader",
         "Talks to the leader and deputy about war",
         "Ignores the low herb stocks, instead practicing battle moves"
     ],
     "bold": [
-        "Decides to try a new herb as treatment for an injury"
+        "Decides to try a new herb as treatment for an injury",
+        "Tries to sneak into a hunting patrol"
     ],
     "calm": [
-        "Stays composed when treating a severe injury"
+        "Stays composed when treating a severe injury",
+        "Didn’t flinch at the smell of a rotting wound",
+        "Reassures r_c that {PRONOUN/r_c/subject} {VERB/r_c/are/is} perfectly fine",
+        "Is telling a story to keep a hurt kitten calm while {PRONOUN/m_c/subject} {VERB/m_c/are/is} treating the kitten’s wounds"
     ],
     "careful": [
-        "Counts and recounts the poppy seeds"
+        "Counts and recounts the poppy seeds",
+        "Overdid it on cleaning a wound again"
     ],
     "charismatic": [
-        "Is doing their daily checkup on the elders"
+        "Is doing {PRONOUN/m_c/poss} daily checkup on the elders",
+        "Makes r_c feel better with m_c’s jokes"
     ],
     "childish": [
-        "Bounces excitedly at the half-moon meeting"
+        "Bounces excitedly at the half-moon meeting",
+        "Is using silly names for the herbs again"
     ],
     "cold": [
-        "Refuses to treat an injured, abandoned kit"
+        "Threatens to make a kit sick when they won’t stay still"
```

Comment:
got another non-ascii apostrophe here

Reply:
OMG I didn't see this and was like running through finding these myself because I noticed, tysm though!! pretty sure I finally got them all<3

Reply:
update: I did not get all of them there's even more, one minute lmao

Reply:
I think I fucked smth up while I fixed those actually?? I'm going insane